0000-rep-template.md

```diff
@@ -0,0 +1,45 @@
+---
+REP Number: <Fill me in with a four-digit unique incrementing number, matching the pull request number; update AFTER REP acceptance.>
```

Comment:
This is confusing.  This should be the pull request number?  If so, "unique incrementing number" doesn't make sense.  So this should be left blank until it's accepted?  Can a REP move from draft immediately to inactive?

Reply:
PR numbers are unique incrementing numbers, but I guess it is a bit confusing to explicitly mention it here. The idea is to left it blank until it's accepted and before merged. Will update this to reflect that.

Reply:
> Can a REP move from draft immediately to inactive?

No, only a PR that tries to add a REP file to the `/active` folder is considered a draft.